### PR TITLE
Deliverable 3

### PR DESCRIPTION
**Pull request by tryfontheo**

### CODE REVIEW

**Comment by huoyanzhizhou** [comment]
There is a red box about the <p> after a <dateline>. Also, many featured divs are sections rather than items, and some items should be put into sections. For example, Local and General on page 3 should be section and Savoy Hotel following Local and General should be inside the Local and General section.

**Comment by whanley** [comment]
Any updates? Pages 5 and 6?

**Comment by tryfontheo** [comment]
Fixes were made and Monday is all done except for one thing I am currently fixing, I will submit this later today. Only other issue that I was encountering when I went back and looked at these sections is that a lot of my sections do not have a feature tag.

File: 1907-03-04.xml [hunk 1]
```diff
@@ -2508,20 +2522,3211 @@
...
+                        Company w mid benefi^acoordmgly.</p>
+                    <p>CAPITAL — Si >oe tbe last report the capita f the company has been increased
+                        by th imoe, in terms of the circular of the 27rl March, 1906, o‘116 910
+                        phare*, which wer» f illy subscribed F- r her, »-ince the date of th
+                        accounts the 4^,000 partly-paid shares men - o-md therein h ve been folly
+                        paid up. an h Debentures have also been relearned an -Mncellcd.</p>
+                    <p>STAFF.— The B >aH desire to ex mess th®ir appreciation of the dent services
+                        render-1 by the entire staff of ’he oompanv in Egypt, in he Soudan and in
+                        Lmdon The company> thanks are specially d ie to Mr. Beresford, itb
+                        •O”8ulting engineer, whose experienced super
```
Comment (huoyanzhizhou) [inline 1907-03-04.xml]: Did you update your measures in this part?

**Comment by huoyanzhizhou** [comment]
Overall good job. Where are your files for other dates? If you are waiting this to be merged befreo you submit further works, please consider this to be merged @whanley

**Comment by whanley** [comment]
Good work on Monday.

**Pull request merged**